jsonpatch.py

```diff
@@ -222,6 +222,9 @@ def __init__(self, patch):
             'copy': CopyOperation,
         }
 
+        for op in self.patch:
+            self._get_operation(op)
+
```

Comment:
Could you add a comment here? It's obvious what it does in the context of the pull request, but without that context it will not be so obvious anymore.